src/Components/WebAssembly/Build/src/buildTransitive/netstandard1.0/Microsoft.AspNetCore.Components.WebAssembly.Build.targets

```diff
@@ -0,0 +1,4 @@
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)..\..\targets\All.targets" Condition="'$(ReferencesComponentsWebAssemblyBuild)' == 'true'" />
```

Comment:
If you have a buildTransitive directory, NuGet ignores the `build/${PackageId}.targets` entirely. The trick used here is to rely on a property imported by `build/${PackageId}.props` that's only available to a project that references the WebAssembly.Build package.

Reply:
That said, the property is kinda useful for future integration with SDK purposes